docs/reference/server-config.md

```diff
@@ -359,6 +361,29 @@ Example:
 	</tr>
 </table>
 
+## Hot logging level reload
+We don't support to completely reload the whole configuration yet at present. What we support for now is:
+- increase logging level by signaling `SIGUSR1`
+- decrease logging level by signaling `SIGUSR2`
+
+Example:
+
+To increase logging level
+```
+$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```

Comment:
Do you think it make sense we should give out the shell command to do this ,like
```
$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```